src/main/resources/hudson/plugins/release/pipeline/ReleaseStep/DescriptorImpl/parameters.groovy

```diff
@@ -0,0 +1,46 @@
+package hudson.plugins.release.pipeline.ReleaseStep.DescriptorImpl
+
+import hudson.model.AbstractProject
+import hudson.model.BuildableItemWithBuildWrappers
+import hudson.model.StringParameterDefinition
+import hudson.plugins.release.ReleaseWrapper;
+def st = namespace('jelly:stapler')
+def l = namespace('/lib/layout')
+l.ajax {
+    def jobName = request.getParameter('job')
+    if (jobName != null) {
+        def contextName = request.getParameter('context')
+        def context = contextName != null ? app.getItemByFullName(contextName) : null
+        def project = app.getItem(jobName, context, AbstractProject)
```

Comment:
This line is not 100% correct since it causes a "No item" error for Pipeline jobs. I will fix it in a follow-up